.github/workflows/ci.yml

```diff
@@ -66,7 +66,9 @@ jobs:
         uses: scottbrenner/cfn-lint-action@v2
 
       - name: Run verification
-        run: make check
+        run: |
+          echo "::add-matcher::.github/actionlint-matcher.json"
```

Comment:
Shouldn't we use `GITHUB_ACTION_PATH` here?

Reply:
I'll try it out. I should work.

Reply:
Hmm. Using `GITHUB_ACTION_PATH` does not work as the var is not available. Accoding to the docs: *This property is only supported in composite actions.* Reverting back to the relative path.